plugins/vlc-video/vlc-video-plugin.h

```diff
@@ -1,16 +1,16 @@
 #include <obs-module.h>
-#include <libvlc.h>
+#include <vlc/libvlc.h>
```

Comment:
for we internally libvlc.h includes stuff with a vlc/ subfolder. I could not find a way to get this working without changing the includes here as well.